### DISCUSSION TITLE
Which way to use the commands for all downloads to run in sequence, similar to how it is done in game updates.

### DISCUSSION THREAD

**Discussion by sandrostoregames** [aldostools/webMAN-MOD#921] [Q&A]
I'm not able to start a download right after another one finishes, I don't know if I'm doing it wrong I need some help.  For example:  /xmb.ps3/download.ps3?to=dev_hdd0/packages&url=http://exemple.com/arquive.pkg;/http://exemple.com;/http://exemple.com/arquive.pkg;/

**Comment by aldostools** [comment]
That feature is not supported. You could try creating a batch script with multiple downloads, to check if that works.
